src/Components/Server/src/Circuits/RemoteJSDataStream.cs

```diff
@@ -161,7 +161,11 @@ public override long Position
     }
 
     public override void Flush()
-        => throw new NotSupportedException();
+    {
+        // No-op
```

Comment:
```suggestion
        // No-op - Flush is specifically allowed on read-only streams.
```

Reply:
Do you want me to update all of these, or are you not fussed with spinning CI to update the comments?

Reply:
I can live without the comment(s).